cli/flox/doc/manifest.toml.md

```diff
@@ -256,13 +256,31 @@ SERVER_PORT = "3000"
 
 The `[hook]` section of the manifest allows you to specify a script that's
 executed immediately after the environment is activated.
-Since the hook runs after activation the environment variables in the `[vars]`
+Since the hook runs after activation, the environment variables in the `[vars]`
 section may be referenced within the hook.
 
 Common usages for environment hooks are printing usage messages or performing
 setup operations such as initializing a database or starting a server.
 
-The script is set using the `script` option.
+### `on-activate`
+The `on-activate` script is run non-interactively in a Bash subshell after the
+environment is activated.
+This is useful for environment initialization that you want done in a consistent
+shell so that you don't need to worry about shell compatibility.
+The exit code and `stdout` of this script are discarded.
+
+```toml
+[hook]
+on-activate = """
+    mkdir -p data_dir
+"""
+```
+
+
+### `script`
```

Comment:
`hook.script` is being deprecated, right?  Unless you plan to have a quick followup PR, it might be good to at least remove the docs for this comment the code that handles it to avoid confusion.

Reply:
https://github.com/flox/flox/pull/1124#issuecomment-1981739737